ast/builtins.go

```diff
@@ -1613,6 +1614,19 @@ var WalkBuiltin = &Builtin{
 	),
 }
 
+// Reachable computes the set of reachable nodes in the graph from a set
+// of starting nodes.
+var Reachable = &Builtin{
```

Comment:
nit: Can we rename the variable to like `ReachableBuiltin`, `GraphReachable`, or similar. As is for the golang API we have `ast.Reachable` which we might want to reserve for other `ast` API's.

Reply:
My preference would be "ReachableBuiltin" like did with "WalkBuiltin" (because ast.Walk already exists.)